opencdms/utils/seeder.py

```diff
@@ -0,0 +1,192 @@
+from datetime import datetime,timedelta
+from uuid import uuid4
+
+from sqlalchemy import create_engine, schema
+from sqlalchemy.orm import sessionmaker, close_all_sessions, Session, clear_mappers
+from faker import Faker
+
+from opencdms.utils.db import get_cdm_connection_string
+from opencdms.provider.opencdmsdb import mapper_registry, start_mappers
+from opencdms.models import cdm
+
+DB_URL = get_cdm_connection_string()
+db_engine = create_engine(DB_URL)
+Base = mapper_registry.generate_base()
+
+def db_session():
+    Session = sessionmaker(bind=db_engine)
+    session = Session()
+    yield session
+    session.close()
+
+def setup():
+    schemas = {v.schema for k, v in Base.metadata.tables.items()}
+
+    for _schema in schemas:
+        if not db_engine.dialect.has_schema(db_engine, _schema):
+            db_engine.execute(schema.CreateSchema(_schema))
+    Base.metadata.create_all(bind=db_engine)
+    start_mappers()
+
+
+def seed_observations(db_session: Session):
+    feature_type = cdm.FeatureType( \
+        name="Feature1", \
+            description="A type of feature", \
+                links=["https://links.features.com/1"]
+            )
+    user = cdm.User(
+        id=str(uuid4()),
+        name="John Doe"
+    )
+    status = cdm.RecordStatus(
+        id=1,
+        name="ACCEPTED",
+        description="Valid record"
+        )
+    
+    time_zone = cdm.TimeZone(abbreviation="UTC",name="lagos/africa",offset="1")
```

Comment:
Should this be UTC+1 or WAT (west African Time) for the abbreviation?

Reply:
Thanks @david-i-berry , Its corrected now.